src/PolyhedralGeometry/Polyhedron/properties.jl

```diff
@@ -832,10 +833,11 @@ Compute the $h^*$ polynomial of `P`.
 # Examples
 ```jldoctest
 julia> c = cube(3)
-A polyhedron in ambient dimension 3
+Polyhedron in ambient dimension 3
 
 julia> h_star_polynomial(c)
 x^3 + 23*x^2 + 23*x + 1
+```
```

Comment:
Off-topic, but: That's the second or third time I've recently seen this kind of issue (missing or misplaced triple backquote). I totally can see this happening to me as well, and I wonder if there is some tool or check we can activate to find these systematically?